py/strato/common/log/lineparse.py

```diff
@@ -21,8 +21,13 @@ def seperateTimestamp(message, timestampFormat=DEFAULT_TIMESTAMP_REGEX):
 
 
 def translateToEpoch(timeStamp, timestampFormat=DEFAULT_TIMESTAMP_FORMAT):
+    ms = 0
+    if timestampFormat.endswith('%f'):  # handle milliseconds
+       separator = timestampFormat[-3]
```

Comment:
is this the only way to get the separator? you cant get it from some variable that is set during config time, or some other way?